### ISSUE TITLE
Data Mapper pattern

### ISSUE THREAD

**Issue by iluwatar** [iluwatar/java-design-patterns#1291]
https://en.wikipedia.org/wiki/Data_mapper_pattern
https://www.sourcecodeexamples.net/2018/04/data-mapper.html

**Comment by gdjs2** [comment]
May I work on this?

**Comment by ohbus** [comment]
⭐ Sure thing! Thanks for your interest in our project 😃 

✔️ Please mention a 📆 timeline 🕙 for when can we expect 🤔 a Pull Request against this issue.

**Comment by ohbus** [comment]
Yes I see, might have been an error by not updating the issues correctly and managing them.

ut it would still need some improvement to **refactor** the **Readme** for this pattern to make it like the new design.

Will you be willing t work on improving this? Please do let me know of the same.
